monkey/monkey_island/cc/ui/src/styles/pages/ConfigurationPage.scss

```
@@ -49,6 +49,10 @@
   font-size: 1.2em;
 }
 
+.config-form .form-group .field-string > p {
```

Is this selecting the correct thing? Have you tried it this way? 
```suggestion
.config-form.form-group.field-string > p {
```

See https://www.w3schools.com/cssref/css_selectors.asp for more information.